### DISCUSSION TITLE
Using GovreadyQ to Author SSPs and Export OSCAL

### DISCUSSION THREAD

**Discussion by ohsh6o** [18F/Security-Compliance#11] [Show and tell]
Hi @its-a-lisa-at-work, given previous conversations in #1 and elsewhere, I ran into @gregelin and think we would really enjoy discussing how to use recent developments to [GovReadyQ](https://github.com/GovReady/govready-q) and build components and combine them into a system security plan. Both components and complete SSPs offer new functionality to export OSCAL XML and JSON.

Should we set something up?

**Comment by its-a-lisa-at-work** [comment]
Awesome, great suggestion -- me and @pburkholder reached out to @gregelin yesterday and would love for him to present at our guild as soon as possible!

**Comment by gregelin** [comment]
@its-a-lisa-at-work @ohsh6o 
Where do we sign up? 
Best for us is after Feb 3...

**Comment by pburkholder** [comment]
How about Feb 12 13:00 EST? @gregelin

**Comment by gregelin** [comment]
@pburkholder Friday 2/12 at 1:00 PM EST is perfect. Send an invite with details, please?

**Comment by pburkholder** [comment]
There should be one in your mailbox, @gregelin

On Wed, Jan 27, 2021 at 1:58 PM Greg Elin <notifications@github.com> wrote:

> @pburkholder <https://github.com/pburkholder> Friday 2/12 at 1:00 PM EST
> is perfect. Send an invite with details, please?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/Security-Compliance/discussions/11#discussioncomment-315638>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJHWCUQWXUHBLF2OTM2HFDS4BO6HANCNFSM4VBX746Q>
> .
>


-- 
-
*Peter Burkholder |  **cloud.gov <https://cloud.gov> compliance & security*
please use cloud-gov-compliance@gsa.gov for cloud.gov matters

*202-709-2028 <(202)%20209-2028> | peter.burkholder@gsa.gov
<peter.burkholder@gsa.gov> *

*| pronouns he-him <https://www.mypronouns.org/he-him>*
*Free/Busy Calendar
<https://calendar.google.com/calendar/embed?src=peter.burkholder@gsa.gov>*

**Reply by ohsh6o** [reply]
Is this actually happening tomorrow, @pburkholder?

**Comment by pburkholder** [comment]
Yes, it is.

On Thu, Feb 11, 2021 at 4:10 PM Alexander Stein <notifications@github.com>
wrote:

> Is this actually happening tomorrow, @pburkholder
> <https://github.com/pburkholder>?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/Security-Compliance/discussions/11#discussioncomment-360919>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJHWCXEDHMUWRD5F4PLJG3S6RBVPANCNFSM4VBX746Q>
> .
>


-- 
-
*Peter Burkholder |  **cloud.gov <https://cloud.gov> compliance & security*
please use cloud-gov-compliance@gsa.gov for cloud.gov matters

*202-709-2028 <(202)%20209-2028> | peter.burkholder@gsa.gov
<peter.burkholder@gsa.gov> *

*| pronouns he-him <https://www.mypronouns.org/he-him>*
*Free/Busy Calendar
<https://calendar.google.com/calendar/embed?src=peter.burkholder@gsa.gov>*
